cmd/provisioner/main.go

```diff
@@ -70,11 +70,11 @@ var leaseRetryPeriod = flag.Duration(
 	"How often lease acquisition and renewal should be retried",
 )
 
-var leaseTermLimit = flag.Duration(
-	"leaseTermLimit",
-	10*time.Minute,
-	"Maximum time that a provisioner can maintain a lease",
-)
+//var leaseTermLimit = flag.Duration(
```

Comment:
Bhagya is this unused declaration? if so can this be removed.

Reply:
yes this is unused declaration. Verified it. kept it commented just incase if this is required in scenarios.